packages/dialtone-icons/package.json

```diff
@@ -6,18 +6,17 @@
     "build": "gulp"
   },
   "files": [
-    "dist"
+    "dist",
+    "vue2/dist",
+    "vue3/dist"
   ],
   "exports": {
-    "./vue2": {
-      "import": "./dist/dialtone-icon@2.js",
-      "require": "./dist/dialtone-icon@2.cjs"
-    },
-    "./vue3": {
-      "import": "./dist/dialtone-icon@3.js",
-      "require": "./dist/dialtone-icon@3.cjs"
-    },
-    "./*": "./dist/*"
+    "./vue2": "./vue2/dist/dialtone-icons.js",
+    "./vue3": "./vue3/dist/dialtone-icons.js",
+    "./vue2/*": "./vue2/dist/components/*.js",
+    "./vue3/*": "./vue3/dist/components/*.js",
+    "./svg/*": "./dist/svg/*",
+    "./*.json": "./dist/*.json"
```

Comment:
Should we still allow `"./*": "./dist/*"` ?

Reply:
No, `"./svg/*"` and `"./*.json"` are covering the exports on dist and `"./vue2/*"` and `"./vue3/*"` are covering the tree-shake on vue 2 and vue 3 icon versions.